### DISCUSSION TITLE
MythTV wiki article created

### DISCUSSION THREAD

**Discussion by JanCeuleers** [bengtmartensson/IrScrutinizer#481] [General]
Just letting you know that I have created an article on the MythTV wiki that refers to your work.

https://www.mythtv.org/wiki/Recording_from_HDMI

**Comment by bengtmartensson** [comment]
Hi Jan,
thank you for the article and the link, and please excuse me for not answering sooner. Nice.

Since you are using features not yet found in  the official release, you should say that the features are (presently) only found in the snapshot releases. Just as a suggestion... (Next official release will probably be in a few months time.)

**Comment by JanCeuleers** [comment]
On 27/02/2023 17:59, Bengt Martensson wrote:
>
> Hi Jan,
> thank you for the article and the link, and please excuse me for not
> answering sooner. Nice.
>
> Since you are using features not yet found in the official release,
> you should say that the features are (presently) only found in the
> snapshot releases. Just as a suggestion... (Next official release will
> probably be in a few months time.)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bengtmartensson/IrScrutinizer/discussions/481#discussioncomment-5125712>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC2CDD7CH2HA7D7YQQOPGHDWZTMQFANCNFSM6AAAAAAVAH5TSE>.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
Done; thank you
